SwiftBar/AppDelegate.swift

```diff
@@ -7,8 +7,20 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         //Instance of Plugin Manager must be created after app launch
         pluginManager = PluginManager.shared
-        if Preferences.shared.pluginDirectoryPath == nil {
-            App.changePluginFolder()
+        while Preferences.shared.pluginDirectoryPath == nil {
+            let alert = NSAlert()
+            alert.messageText = "Set SwiftBar Plugins Location"
+            alert.informativeText = "Select a folder to store the plugins repository"
```

Comment:
I suggest to change it to "Select a folder to store the SwiftBar plugins" because in a sense this is what this folder is for.